src/Sulu/Bundle/CustomUrlBundle/Resources/public/js/components/webspace/settings/custom-url/main.js

```diff
@@ -0,0 +1,243 @@
+                                attribute: 'changed',
+                                name: 'changed',
+                                content: this.translations.changed,
+                                type: 'datetime'
+                            },
+                            {
+                                attribute: 'changerFullName',
+                                name: 'changerFullName',
+                                content: this.translations.changer
+                            },
+                            {
+                                attribute: 'created',
+                                name: 'created',
+                                content: this.translations.created,
+                                type: 'datetime'
+                            },
+                            {
+                                attribute: 'creatorFullName',
+                                name: 'creatorFullName',
+                                content: this.translations.creator
+                            }
+                        ]
+                    }
+                }
+            ]);
+        },
+
+        /**
+         * Start edit overlay for given id.
+         *
+         * @param {Integer} id
+         */
+        edit: function(id) {
+            this.sandbox.start([
+                {
+                    name: 'webspace/settings/custom-url/overlay@sulucustomurl',
+                    options: {
+                        el: '#webspace-custom-url-form-overlay',
+                        id: id,
+                        webspace: this.data,
+                        saveCallback: this.save.bind(this),
+                        translations: this.translations
+                    }
+                }
+            ]);
+        },
+
+        /**
+         * Deletes selected records from datagrid after asking for confirmation.
+         */
+        delete: function() {
+            var ids = this.sandbox.util.deepCopy($('#webspace-custom-url-list').data('selected'));
+
+            this.sandbox.sulu.showDeleteDialog(function(confirmed) {
+                if (!!confirmed) {
+                    this.sandbox.util.save(
+                        this.templates.url({webspace: this.data, id: null, ids: ids}),
+                        'DELETE'
+                    ).then(function() {
+                        for (var i = 0, length = ids.length; i < length; i++) {
+                            var id = ids[i];
+                            this.sandbox.emit('husky.datagrid.record.remove', id);
+                        }
+                    }.bind(this));
+                }
+            }.bind(this));
+        },
+
+        /**
+         * Save record identified by id with given data.
+         * If id is null a new record will be created.
+         *
+         * @param {Integer} id
+         * @param {{}} data
+         */
+        save: function(id, data) {
+            return this.sandbox.util.save(
```

Comment:
Wouldn't you introduce a `CustomUrlManager` for that, corresponding to the managers in some other bundles?